ci/build.ps1

```diff
@@ -0,0 +1,125 @@
+Set-PSDebug -Trace 1
```

Comment:
I added this because it causes each line to be printed as it's executed, similar to `echo on` in Batch, except it doesn't print the arguments used when calling other executables.